infra/development/base/main.tf

```diff
@@ -0,0 +1,40 @@
+terraform {
```

Comment:
Skip reviewing any file in the infra/development directory, as some of it duplicates code in the infra/production folder. I've added comments to explain the implementation.